include/deal.II/lac/utilities.h

```diff
@@ -50,6 +51,26 @@ namespace Utilities
                                       const VectorType &v0,
                                       const unsigned int k,
                                       VectorMemory<VectorType> &vector_memory);
+
+    /**
+     * Apply Chebyshev filter of @p op with a given positive @p degree
+     * to the input subspace @p x . @p a and @p b define the unwanted
+     * spectrum $ [a,b] $. @p vector_memory is used to allocate memory for
+     * temporary objects. Chebyshev polynomial will be scaled to unity at
+     * @p a_L to prevent overflow. If @p a_L equals to
+     * <code>numbers::signaling_nan<double>()</code>, no scaling is performed.
```

Comment:
@bangerth `std::numeric_limits` does not have `nan`, it has `quite_nan` and `signaling_nan`. How about using `std::numeric_limits::infinity`? that's essentially what we do in ` numbers::invalid_unsigned_int`.